src/main/resources/podcasts.csv

```diff
@@ -0,0 +1 @@
+id,title,author,(episodeId)
```

Comment:
There should be a newline at the end of the header.

Reply:
I'll make those changes now, for this file as well as the users.csv.